# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/cairo/cairo/src/cairo-features.h
# Commit: 61ef06598b0c
# Full Hash: 61ef06598b0c87016b6eecabf22cde2b7f1fbeb6
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2024-05-07 09:13:07
# Regressor Bug: 1892913
# File Overlap Count: 2
# Description:
#   Bug 1892913 - patch 18 - Cairo fixes for iOS build. r=gfx-reviewers,lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D209426
# ==============================================================================

diff -r 51b400f880e3 -r 61ef06598b0c gfx/cairo/cairo/src/cairo-features.h
--- a/gfx/cairo/cairo/src/cairo-features.h	Mon May 06 19:25:20 2024 +0000
+++ b/gfx/cairo/cairo/src/cairo-features.h	Mon May 06 19:25:20 2024 +0000
@@ -75,6 +75,10 @@
 #define CAIRO_HAS_QUARTZ_APPLICATION_SERVICES 1
 #endif
 
+#if defined(MOZ_WIDGET_UIKIT)
+#define CAIRO_HAS_IMAGE_IO 1
+#endif
+
 #ifdef XP_WIN
 #define CAIRO_HAS_DWRITE_FONT 1
 #define CAIRO_HAS_WIN32_FONT 1